tests/store/test_store_api.py

```diff
@@ -1,8 +1,12 @@
 """Tests for the store api"""
 import datetime
+from typing import List
 
+from housekeeper.store.models import File
 
-def test_fetch_bundles(populated_store, bundle_data_old):
+
+
+def test_fetch_bundles(populated_store, bundle_data_old, time_stamp_now):
     """
     test fetch all files when two bundles are added
```

Comment:
```suggestion
    Test return all files when two bundles are added.
```

Reply:
Fixed but please don't comment on arbitrary code in the code base to fix.